strings/base_windows.h

```diff
@@ -302,7 +291,7 @@ WINRT_EXPORT namespace winrt
     }
 
     template <typename T, std::enable_if_t<!std::is_base_of_v<Windows::Foundation::IUnknown, T>, int> = 0>
-    auto put_abi(T& object) noexcept
+    auto put_abi(T & object) noexcept
```

Comment:
What's happening here? Earlier in this same file, it left these reference parameters left-aligned. But here it suddenly switches to middle alignment?

Reply:
Not sure about this one.  I tried setting `ReferenceAlignment: Left` and it still adds the space.

Reply:
I think the templated T is what confuses it.  I see _many_ examples of changes in this PR where `var & i` is correctly adjusted to be `var& i`.  So this is not universally wrong.  There are just a small number of false positive type matches.